examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/ExampleScenarioFactory.java

```diff
@@ -86,10 +91,29 @@ private FrameworkExampleScenario getFrameworkScenario(final String framework) {
     public Map<String, String> getResourceTemplateMap() {
         Map<String, String> result = new HashMap<>();
         for (FeatureExampleScenario each : featureScenarios) {
-            result.putAll(each.getJavaClassTemplateMap());
+            result.putAll(each.getResourceTemplateMap());
         }
         result.putAll(frameworkScenario.getResourceTemplateMap());
         result.put("resources/logback.ftl", "logback.xml");
         return result;
     }
+    
+    public Set<String> getJavaClassPathSet() {
+        Set<String> result = new HashSet<>();
```

Comment:
Use Collection to instead of Set if passable, upper layer of interface should be used in first period.